pkg/testutil/test/command.go

```diff
@@ -90,6 +96,16 @@ func (gc *GenericCommand) Run(expect *Expected) {
 	} else {
 		iCmdCmd := gc.boot()
 		env = iCmdCmd.Env
+
+		if gc.pty {
+			pty, tty, _ := Open()
+			iCmdCmd.Stdin = tty
+			iCmdCmd.Stdout = tty
+			iCmdCmd.Stderr = tty
+			defer pty.Close()
+			defer tty.Close()
+		}
```

Comment:
The pty variable is just closed without using it?

Reply:
It is just the handle of /dev/ptmx - only purpose is to get a new /dev/pts/x